variables.tf

```
@@ -20,6 +20,12 @@ variable "node_type" {
   type        = string
 }
 
+variable "settings" {
+  description = "Database settings configuration"
+  type        = map(string)
+  default     = {}
```

The default value override Scaleway's one. Please set it to `null` instead
```suggestion
  default     = null
```